src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/TrafficSeo/Meta/ShopUrlType.php

```diff
@@ -59,8 +60,13 @@ class ShopUrlType extends AbstractType
      * @param bool $isShopFeatureActive
      * @param bool $doesMainShopUrlExist
      */
-    public function __construct($isHostMode, $isShopFeatureActive, $doesMainShopUrlExist)
-    {
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales, $isHostMode,
```

Comment:
```suggestion
        array $locales,
        $isHostMode,
```

You'll need to update the phpdoc too ;)